### PR TITLE
Uses always the first tab on new FI

### PR DESCRIPTION
**Pull request by mholthausen**
This MR update the `FeatureInfo` to use always the first layer tab when a new feature info is triggered

@terrestris/devs please review

### CODE REVIEW

File: src/components/ToolMenu/FeatureInfo/index.tsx [hunk 1]
```diff
@@ -260,6 +280,13 @@ export const FeatureInfo: React.FC<FeatureInfoProps> = ({
       serializedFeatures[layerName] = new OlFormatGeoJSON().writeFeatures(selectedFeatures);
     });
 
+    const layers: MapLayer[] = Object.keys(features).map(layerName => ({
+      layerName: layerName,
+      index: findMapLayerIndex(layerName)
+    })).sort((a, b) => b.index - a.index);
+
+    setActiveTabKey(layers[0].layerName);
```
Comment (dnlkoch) [inline src/components/ToolMenu/FeatureInfo/index.tsx:288]: What happens if `layers[0]` is not available?

Comment (mholthausen) [reply src/components/ToolMenu/FeatureInfo/index.tsx]: A check is added here to test if layers is present and has at least one value.

File: src/components/ToolMenu/FeatureInfo/index.tsx [hunk 2]
```diff
@@ -122,6 +128,27 @@ export const FeatureInfo: React.FC<FeatureInfoProps> = ({
     return <></>;
   }
 
+  const changeActiveKey = (key: string) => {
+    setActiveTabKey(key);
+  };
+
+  const findMapLayerIndex = (layerName: string) => {
```
[the remark this answers is not in the document]
Comment (dnlkoch) [reply src/components/ToolMenu/FeatureInfo/index.tsx:135]: Yes, you're right. Never mind.

**Review by dnlkoch — APPROVED** [review]

**Pull request merged**